pajbot/managers/emote.py

```diff
@@ -220,6 +220,9 @@ def twitch_emote(emote_id: str, code: str) -> Emote:
                 "2": EmoteManager.twitch_emote_url(emote_id, "2.0"),
                 "4": EmoteManager.twitch_emote_url(emote_id, "3.0"),
             },
+            # Twitch emotes are always size-fixed
```

Comment:
Technically not true

Reply:
here are some still available wide emotes for testing, if needed: https://emotes.raccatta.cc/twitch/thegdstudio